tests/__snapshots__/LaravelEnumTransformerTest__test_it_can_transform_an_enum__1.txt

```diff
@@ -1,5 +1,4 @@
-export enum Enum {
+
```

Comment:
Just this feels a little odd here with the export being removed.

Are we also just able to add a test to the different type as well?

Reply:
Yea it's a strange change chnage, adding the type is now handled by the `spatie/typescript-transformer` library rather than this transformer.

I'll look at adding separate tests for the type and enum conversion

Reply:
Ok, if that is the case we can just merge to get thing progressing